core/dump.c

```diff
@@ -340,3 +340,70 @@ void dump_vmcs(struct vcpu_t *vcpu)
         vmread_dump(vcpu, enc, name);
     }
 }
+
+void dump_vmx_info(struct info_t *info)
+{
+    hax_log("VMCS Rev %d\n", info->_vmcs_revision_id);
+
+    hax_log("VMX basic info       : 0x%016llX\n",
+            info->_basic_info);
+    hax_log("VMX misc info        : 0x%08llX\n",
```

Comment:
Would `08` truncate the upper 32 bits? According to Intel SDM:

> Bits 63:32 report the 32-bit MSEG revision identifier used by the processor.

Not sure how useful this information is, but `016` would be a safe choice.

Reply:
For the record: No truncation occurs ([check Ideone snippet](https://ideone.com/uKfC8r)), but I still agree that if those upper bytes are used, padding to 32-bits makes no sense and `%016llX` is much more convenient.

Reply:
That's good to know, thanks!